src/TeamStore.DevUI/wwwroot/css/site.css

```diff
@@ -36,7 +36,48 @@
     border: none;
 }
 
-@media all and (min-width:1200px){
+ul.dropdown-menu.invisible-asset-holder {
```

Comment:
Several things here.
If the CSS is global (not part of a @media definition) move it outside of the @media section
although not very apparent, CSS classes should have our ob-* prefix.
In general, CSS could follow the BEM naming convention.

Reply:
Oops, just saw that CSS is outside @media